homeassistant/components/media_player/plex.py

```diff
@@ -114,10 +119,15 @@ def setup_plexserver(host, token, hass, add_devices_callback):
     plex_clients = {}
     plex_sessions = {}
 
-    @util.Throttle(MIN_TIME_BETWEEN_SCANS, MIN_TIME_BETWEEN_FORCED_SCANS)
```

Comment:
I removed the Throttle and added this instead:
```
track_utc_time_change(
        hass, update_devices,
        second=range(0, 60, MIN_TIME_BETWEEN_SCANS.seconds)
    )
```
This is still in the `setup_plexserver` function, so it should only run once per instance.
I replaced it because I noticed that by doing this, afterwards the clients were discovered even if there wasn't an active one when HA started.